src/files.rs

```diff
@@ -136,6 +137,21 @@ impl Files {
     Ok(Response::new(Body::from(body.into_string())))
   }
 
+  fn download_icon(size: u8) -> PreEscaped<String> {
+    html! {
+    svg
+      width=(size)
```

Comment:
Apparently you can also set these attributes with CSS:

```css
.icon {
  width: 24px;
  height: 24px;
  stroke: currentColor;
  stroke-width: 2;
  stroke-linecap: round;
  stroke-linejoin: round;
  fill: none;
}
```

This isn't that important now, but if we have more than one icon, this would allow us to avoid repeating attributes on each icon, but using, .e.g., `.icon` / `.icon-large`, etc.

Reply:
Another benefit is that the HTML will be smaller, because there won't be repeated attributes.

Reply:
Interesting. I'm switching to that.

I think there's also some advantages to trying to use Rust's abstraction mechanisms as much as possible. (And ignoring duplication in the html.) But yeah, it's a bit too early to see how this should actually be done. For one icon it certainly doesn't matter much.